src/effects/builtin/whitenoiseeffect.h

```diff
@@ -0,0 +1,43 @@
+#ifndef WHITENOISEEFFECT_H
+#define WHITENOISEEFFECT_H
+
+#include "effects/effectprocessor.h"
+#include "engine/effects/engineeffect.h"
+#include "engine/effects/engineeffectparameter.h"
+#include "engine/filters/enginefilterpansingle.h"
+#include "util/class.h"
+#include "util/defs.h"
+#include "util/sample.h"
+#include "util/types.h"
+
+class WhiteNoiseGroupState final : public EffectState {
+  public:
+    WhiteNoiseGroupState(const mixxx::EngineParameters& bufferParameters)
+      : EffectState(bufferParameters) {
+    }
+    ~WhiteNoiseGroupState() {
+    }
+};
+
+class WhiteNoiseEffect : public EffectProcessorImpl<WhiteNoiseGroupState> {
+  public:
+    WhiteNoiseEffect(EngineEffect* pEffect);
+    virtual ~WhiteNoiseEffect();
+
+    static QString getId();
+    static EffectManifestPointer getManifest();
+
+    // See effectprocessor.h
+    void processChannel(const ChannelHandle& handle,
+                        WhiteNoiseGroupState* pState,
+                        const CSAMPLE* pInput, CSAMPLE* pOutput,
```

Comment:
This should be wrapped. Aren't you using our [pre-commit hooks](https://github.com/mixxxdj/mixxx/wiki/Using-Git#set-up-automatic-code-checking)?

Reply:
I will look into that